src/django/api/processing.py

```diff
@@ -298,6 +317,20 @@ def make_pending_match(item_id, facility_id, score):
                     'one_gazetteer_match_greater_than_threshold'
                 item.status = FacilityListItem.MATCHED
                 item.facility = matches[0].facility
+            elif len(quality_matches) > 1:
+                exact_matches = [m for m in quality_matches
+                                 if is_string_match(item, m.facility)]
+                # We check == 1 because multiple exact matches should not
+                # happen. They are an indication of duplicate facility data
+                # that should be merged through moderation tools. Showing the
+                # multiple potential matches to the contributor increases the
+                # visibility of the issue.
+                if len(exact_matches) == 1:
+                    exact_matches[0].status = FacilityMatch.AUTOMATIC
+                    exact_matches[0].results['match_type'] = \
+                        'multiple_gazetteer_matches_with_one_exact_string_match'
```

Comment:
The length of this line is causing a CI failure:

```
./api/processing.py:331:80: E501 line too long (80 > 79 characters)
```

Consider adding an ignore directive to exempt it:

```suggestion
                        'multiple_gazetteer_matches_with_one_exact_string_match'  NOQA
```

Reply:
Thanks. Fixed in the rebase.